### ISSUE TITLE
Would be cool if there would be a few default templates

### ISSUE THREAD

**Issue by bassbump** [mjmlio/mjml#561]
Non branded, similar to http://foundation.zurb.com/emails/email-templates.html

So beginners can get started very quickly

**Comment by ngarnier** [comment]
hey @bassbump, have you seen https://mjml.io/templates? 

We had more basic templates but removed them because we thought they didn't bring much value:
- https://mjml.io/try-it-live/templates/kili
- https://mjml.io/try-it-live/templates/mama-mia
- https://mjml.io/try-it-live/templates/underwater

Does it help?

**Comment by bassbump** [comment]
Yeah I saw https://mjml.io/templates, but if that is community sourced it's very limited and gives the impression that it's not widely adopted.
But I tried it anyway and I really like the syntax.

I just think, pure marketing wise, that it would give a good impression if you would provide a few neutral boilerplate templates as well. Just a suggestion, not a must!

**Comment by ngarnier** [comment]
Alright, we've taken an approach to show more real use cases with templates we either created ourselves or from the community to showcase their work but I see your point. 

We'll try to create and add more neutral boilerplate templates, feel free to suggest some too :).

**Comment by Renjamin** [comment]
I'm actually working on a few ideas.  I just migrated to the MJML framework, as of last week, but I'm having lots of fun.  My goal is to create a few templates for the community within a week or two. :)

**Comment by ngarnier** [comment]
Oh that's awesome, looking forward to showcasing your templates @Renjamin :)

**Comment by Renjamin** [comment]
Just a quick follow up on this.  I've been swamped, as I'm sure we all are, but I do have some designs templates I started mocking up, and will be reproducing them in MJML as quickly as I can.

Thanks for everyone's patience. :)
